doc/gc.rst

```diff
@@ -43,17 +43,17 @@ To choose the memory management strategy use the ``--gc:`` switch.
   simply never freed. You should use ``--gc:arc`` instead.
 
 
-================== ======== ================= ======================
-Memory Management  Heap     Reference Cycles  Command line switch
-================== ======== ================= ======================
-RefC               Local    Cycle Collector   ``--gc:refc``
-Mark & Sweep       Local    Cycle Collector   ``--gc:markAndSweep``
-ARC                Shared   Leak              ``--gc:arc``
-ORC                Shared   Cycle Collector   ``--gc:orc``
-Boehm              Shared   Cycle Collector   ``--gc:boehm``
-Go                 Shared   Cycle Collector   ``--gc:go``
-None               Manual   Manual            ``--gc:none``
-================== ======== ================= ======================
+================== ======== ================= ============== ===================
+Memory Management  Heap     Reference Cycles  Stop-The-World Command line switch
```

Comment:
What does "stop the world" mean in this context? One thread, all threads, deterministic vs indeterministic, etc?

Reply:
Unless explicitly specified people usually refer to all the threads of a process. Do you suggest a better naming?.